Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
-# Execute system update
-RUN yum -y update && yum clean all
```

Comment:
Seeing as I didn't get the support, sorry consensus ;), I wanted in e-mail I'll add this back in. You win - for now ;)